tests/plugins/copyformatting/manual/focus.html

```diff
@@ -0,0 +1,41 @@
+
+<textarea name="editor1" id="editor1" cols="30" rows="10">
+	<p>&nbsp;</p>
+	<p>&nbsp;</p>
+	<p>&nbsp;</p>
+	<p>&nbsp;</p>
+	<p>&nbsp;</p>
+	<p>&nbsp;</p>
+	<p>&nbsp;</p>
+	<p>&nbsp;</p>
+	<p>&nbsp;</p>
+	<p>&nbsp;</p>
+	<p>&nbsp;</p>
+	<p>&nbsp;</p>
+	<p>&nbsp;</p>
+	<p>&nbsp;</p>
+	<p>&nbsp;</p>
+	<p>&nbsp;</p>
+	<p>&nbsp;</p>
+	<p>&nbsp;</p>
+	<p>&nbsp;</p>
+	<p>&nbsp;</p>
+	<p>&nbsp;</p>
+	<p>&nbsp;</p>
+	<p>&nbsp;</p>
+	<p>&nbsp;</p>
+	<p><img alt="" src="http://s1.ckeditor.com/themes/ckeditor4/images/slider-cke46-ea0c506c78.png" style="height:63px; width:200px" /></p>
+	<p><img alt="cake" src="http://c.cksource.com/a/1/img/demo/brownie.jpg" style="height:133px; width:200px" /></p>
+
+	<p>&nbsp;</p>
+
+	<div style="float:right; padding:5px; width:210px"><a href="http://ckeditor.com/"><img alt="Focus jumps to the top of the editor" src="http://c.cksource.com/a/1/img/demo/brownie.jpg" style="height:133px; width:200px" /></a></div>
+
+	<p>&nbsp;</p>
+
+	<p>&nbsp;</p>
+</textarea>
+
+<script>
+	CKEDITOR.replace( 'editor1' );
```

Comment:
Again, from what I understand this was the Edge only fix, what was the reason of removing
```
if ( !CKEDITOR.env.ie ) {		
 	bender.ignore();		
 }
```
ignore from this test? Does the changes somehow affect other browsers?

Reply:
The problem was with IE, but older fix corrupted Edge, so I thought it's good idea to test all other browsers.